packages/plugins/src/SearchIndexPlugin.ts

```diff
@@ -86,7 +143,7 @@ const SearchIndexPlugin: PluginType<Page, SearchIndexPluginOptions> = {
   async $beforeSend(
```

Comment:
Interested to know why `$beforeSend` was picked instead of `$afterSource`.  I am trying to get document why you would pick one over the other...